cmd/datasource-syncer/main.go

```diff
@@ -222,3 +238,43 @@ func buildUpdateDataSourceRequest(dataSource grafana.DataSource, token string) (
 	dataSource.SecureJSONData[authHeaderValue] = fmt.Sprintf("Bearer %s", token)
 	return &dataSource, nil
 }
+
+func getTLSClient(logger log.Logger, certFile, keyFile, caFile string, insecureSkipVerify bool) (*http.Client, error) {
+	if (certFile != "" || keyFile != "") && (certFile == "" || keyFile == "") {
+		return nil, fmt.Errorf("--tls-cert and tls-key must both be set or unset")
+	}
+
+	if certFile == "" && keyFile == "" && caFile == "" && !insecureSkipVerify {
+		return http.DefaultClient, nil
```

Comment:
suggestion: the grafana library actually uses https://github.com/hashicorp/go-cleanhttp. Alternatively, just return nil and let the library decide the defaults.